Assets/BossRoom/Scripts/DebugCheats/DebugCheatsManager.cs

```diff
@@ -27,6 +27,20 @@ public class DebugCheatsManager : NetworkBehaviour
         [SerializeField]
         KeyCode m_OpenWindowKeyCode = KeyCode.Slash;
 
+        ServerSwitchedDoor m_ServerSwitchedDoor;
```

Comment:
For debugs/cheats I'm usually not too concerned with caching references like this one. That's to say that using `FindObjectOfType` every time the cheat is called is no big deal, in my opinion. Though it's not a blocker for my review!